include/GL/glui.h

```diff
@@ -581,7 +642,7 @@ class GLUIAPI GLUI_Glut_Window : public GLUI_Node
     void (*glut_mouse_CB)(int,int,int,int);
     void (*glut_visibility_CB)(int);
     void (*glut_motion_CB)(int,int);
-    void (*glut_display_CB)(void);
```

Comment:
Generally I have precious few hours in my life. You can change what you like. I don't understand how to separate changes. In general these are not changes worthy of discussion.

My view is stuff like `void` here is clutter that creates the appearance that there's something going on when there is nothing. It's noise that makes it difficult to assess code. I changed it without giving it much thought. When I see highlight text I expect it to be meaningful.